modules/internal/DefaultAssociative.chpl

```diff
@@ -147,7 +147,43 @@ module DefaultAssociative {
         }
       }
     }
-  
+ 
+    iter these(param tag: iterKind) where tag == iterKind.standalone {
+      if debugDefaultAssoc {
+        writeln("*** In associative domain standalone iterator");
+      }
+      // We are simply slicing up the table here.  Trying to do something
+      //  more intelligent (like evenly dividing up the full slots, led
+      //  to poor speed ups.
+      const numIndices = tableSize;
+      const numChunks = _computeNumChunks(numIndices);
+
+      if debugAssocDataPar {
+        writeln("### numChunks=", numChunks, ", numIndices=", numIndices);
+      }
+
+      if numChunks == 1 {
+        var followThisTab = this.table;
```

Comment:
Nothing.  It came from some checks in the follower that I was able to remove in the standalone case.  I'll remove it. Thanks.